pkg/storage/storage.go

```diff
@@ -0,0 +1,45 @@
+package storage
```

Comment:
I think the `storage` package should in appfile dir which is `pkg/appfile` rather than in pkg dir since it service for appfile.

Reply:
Done